src/core/errors/error.ts

```diff
@@ -9,7 +9,8 @@
 export class GenError extends Error {
     constructor(message: string) {
         super(message);
+        // TODO Set `GenError` prototype because of
         // https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#extending-built-ins-like-error-array-and-map-may-no-longer-work
-        Object['setPrototypeOf'](this, GenError.prototype);
+        // Caution: PhantomJS does not implement `setPrototypeOf`
```

Comment:
Can you instead leave the commented out `setPrototypeOf` and add an `XXX: PhantomJS does not implement setPrototypeOf. Uncomment this when we support tests on headless chrome.`.